Gemfile.lock

```diff
@@ -143,15 +145,15 @@ GEM
     arel (9.0.0)
     ast (2.4.1)
     awrence (1.1.1)
-    aws-eventstream (1.0.3)
-    aws-partitions (1.235.0)
-    aws-sdk-core (3.75.0)
-      aws-eventstream (~> 1.0, >= 1.0.2)
-      aws-partitions (~> 1, >= 1.228.0)
+    aws-eventstream (1.1.0)
+    aws-partitions (1.388.0)
+    aws-sdk-core (3.109.1)
+      aws-eventstream (~> 1, >= 1.0.2)
+      aws-partitions (~> 1, >= 1.239.0)
```

Comment:
heads up, this resulted in some proxy issues in lower envs, we need to double check with @pauldoomgov that devops deploy happens in staging so we don't mess up SMS/voice